falcon_kit/util/system.py

```diff
@@ -18,10 +18,10 @@ def only_these_symlinks(dir2paths):
         base2rel = {os.path.basename(path): os.path.relpath(path, d) for path in paths}
         assert len(base2rel) == len(bases), 'Non-unique basename in {}'.format(repr(paths))
         for existing_base in os.listdir(d):
-            existing_path = os.path.join(d, existing_basename)
+            existing_path = os.path.join(d, existing_base)
```

Comment:
Already fixed on branch `master`. If you have time, you can update this PR like this:
```sh
git pull --rebase
git push -f pb-chin my-branch:minor_type_fix
```